### PR TITLE
Add docs for gctuner in PD

### PR DESCRIPTION
**Pull request by hnes**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

issue: tikv/pd#5827

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/12643
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: pd-configuration-file.md [hunk 1]
```diff
@@ -110,6 +110,48 @@ This document only describes parameters that are not included in command-line pa
...
++ Controls whether to enable the GOGC Tuner.
++ Default value: `false`
+
+### `gc-tuner-threshold` <span class="version-mark">New in v6.6.0</span>
+
+> **Warning:**
+>
+> This configuration is an experimental feature. It is not recommended to use it in production environments.
+
++ The maximum memory threshold ratio for tuning GOGC. When the memory exceeds this threshold, i.e. the value of `server-memory-limit` * `gc-tuner-threshold`, GOGC Tuner stops working.
```
Comment (Oreoxmt) [inline pd-configuration-file.md:150]:
```suggestion
+ The maximum memory threshold ratio for tuning GOGC. When the memory exceeds this threshold, i.e. the value of `server-memory-limit * gc-tuner-threshold`, GOGC Tuner stops working.
```

File: pd-configuration-file.md [hunk 2]
```diff
@@ -110,6 +110,48 @@ This document only describes parameters that are not included in command-line pa
...
++ Minimum value: `0`
++ Maximum value: `0.99`
+
+### `server-memory-limit-gc-trigger` <span class="version-mark">New in v6.6.0</span>
+
+> **Warning:**
+>
+> This configuration is an experimental feature. It is not recommended to use it in production environments.
+
++ The threshold ratio at which PD tries to trigger GC. When the memory usage of PD reaches the value of `server-memory-limit` * `server-memory-limit-gc-trigger`, PD triggers a Golang GC. Only one GC is triggered in one minute.
```
Comment (Oreoxmt) [inline pd-configuration-file.md:130]:
```suggestion
+ The threshold ratio at which PD tries to trigger GC. When the memory usage of PD reaches the value of `server-memory-limit * server-memory-limit-gc-trigger`, PD triggers a Golang GC. Only one GC is triggered in one minute.
```

**Review by Oreoxmt — APPROVED** [review]

File: pd-configuration-file.md (hunk 1 above)
Comment (hnes) [reply pd-configuration-file.md]: Thanks a lot for your comments. I think we could also follow the phrasing in [TiDB's doc](https://docs.pingcap.com/tidb/stable/system-variables#tidb_server_memory_limit_gc_trigger-new-in-v640).

File: pd-configuration-file.md (hunk 2 above)
Comment (hnes) [reply pd-configuration-file.md]: Please see [here](https://docs.pingcap.com/tidb/stable/system-variables#tidb_server_memory_limit_gc_trigger-new-in-v640).

**Pull request merged**